lib/kramdown/converter/syntax_highlighter/rouge.rb

```diff
@@ -27,7 +27,16 @@ def self.call(converter, text, lang, type, call_opts)
       lexer = ::Rouge::Lexer.find_fancy(lang || opts[:default_lang], text)
       return nil if opts[:disable] || !lexer
 
-      formatter = (opts.fetch(:formatter, ::Rouge::Formatters::HTML)).new(opts)
+      begin
+        # Rouge::Formatters::HTMLLegacy is available on Rouge 2.0 or later
+        formatter_class = ::Rouge::Formatters::HTMLLegacy
+        opts[:css_class] ||= 'highlight'
```

Comment:
This is requirement for backward compatibility.

While [Rouge < 2.0's used to add 'highlight' class to parsed codes](https://github.com/jneen/rouge/blob/0415a095744b46bad6c3aed35133458cc1770e2b/lib/rouge/formatters/html.rb#L27), Rouge >= 2.0 adds 'codehilite' by default even if you use `Rouge::Formatters::HTMLLegacy`. So, without specifing an explicit css class, users perhaps encounter regression.